data/templates/frr/bgp.frr.tmpl

```diff
@@ -369,6 +368,11 @@ router bgp {{ asn }}
  exit-address-family
  !
 {%   endif %}
+{%   if parameters.ebgp_requires_policy is defined %}
+ bgp ebgp-requires-policy
+{%   else %}
+ no bgp ebgp-requires-policy
```

Comment:
The `no bgp ebgp-requires-policy` node is also defined here: https://github.com/vyos/vyos-1x/pull/744/files#diff-c8405ef7cc0b5938bde66328e60f83c3cdc8d47f26a59bf86d646a63d1d4e5a7R182 which markes it as duplicated code.

Please change this to:
`{%   if parameters.ebgp_requires_policy is not defined %}no{% endif %} bgp ebgp-requires-policy`

Reply:
The duplication of it was intentional at first. I did not know if the first one (that you listed) would be properly overridden by the removal of an active config. I wanted to absolutely make sure that it wouldn't accidentally be activated. I did a test though, and the original "no" form works even when removing the duplicate on line 374. Therefore I have removed it, yay :) 

I'll look at the second suggestion. I am unsure if the logic would work. Will test now.